GitVersion.yml

```diff
@@ -0,0 +1,4 @@
+next-version: 8.0.0
+branches:
+  main:
+    tag: v8alpha
```

Comment:
I've updated this based on our conversation yesterday, though the `v8` prefix seems redundant to me.